tests/baselines/reference/tsxAttributeResolution3.errors.txt

```diff
@@ -43,17 +49,19 @@ tests/cases/conformance/jsx/file.tsx(39,8): error TS2322: Type '32' is not assig
     // Error
     var obj5 = { x: 32, y: 32 };
     <test1 x="ok" {...obj5} />
-                  ~~~~~~~~~
-!!! error TS2606: Property 'x' of JSX spread attribute is not assignable to target property.
-!!! error TS2606:   Type 'number' is not assignable to type 'string'.
+           ~~~~~~~~~~~~~~~~
+!!! error TS2322: Type '{ x: number; y: number; }' is not assignable to type 'Attribs1'.
+!!! error TS2322:   Types of property 'x' are incompatible.
+!!! error TS2322:     Type 'number' is not assignable to type 'string'.
     
     // OK

```

Comment:
Sorry, I meant *should* be. Right the comment is `// OK` but `<test1 {...obj6} />` actually causes an error.